registry_resolver/src/registry_client.rs

```diff
@@ -1,5 +1,5 @@
 pub mod registry {
-    include!(concat!(env!("OUT_DIR"), "/registry_api.v1.rs"));
+    include!(concat!(env!("OUT_DIR"), "/gen/registry_api.v1.rs"));
```

Comment:
Have we verified that `registry_api.v1.rs` is the right file structure? I'm pretty sure `tonic-build` doesn't produce a `.v1` suffix (infact rust takes issue with full points in rust files other than the one in `.rs`).

Reply:
After doing a clean build:
```
➤ exa -T target/debug/build/registry_resolver-*
target/debug/build/registry_resolver-29dfdb74e79f4851
├── invoked.timestamp
├── out
│  ├── gen
│  │  └── registry_api.v1.rs
│  ├── proto_descriptor.bin
│  └── protos
│     └── registry-mgmt
│        └── registry_api
│           └── v1
│              └── registry.proto
├── output
├── root-output
└── stderr
target/debug/build/registry_resolver-eac1c387d1002808
├── build-script-build
├── build_script_build-eac1c387d1002808
└── build_script_build-eac1c387d1002808.d
```

so it looks like the tonic code is in fact generating a rust source file with a period in it, which doesn't seem to be breaking anything as far as I can tell.